lib/tasks/pexel.rake

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'pexels'
+require './app/services/pexels_service.rb'
+
+namespace :pexel do
+  desc 'TODO'
```

Comment:
Pls make description and name task according it's functionality. Currently I cannot understand what is this doing without checking it's code. Pls deal with it.